backend/migration/utils.py

```diff
@@ -94,7 +95,9 @@ def _recreate_collection(
     src_config = src_collection_info.config
     src_payload_schema = src_collection_info.payload_schema
 
-    dest_client.recreate_collection(
+    # delete destination collection only from qdrant that was created while creating metadatastore entry
+    dest_client.delete_collection(destination_collection_name, timeout=300)
```

Comment:
Is there a `wait` option?

Reply:
don't see any wait. might add sleep?

Reply:
I supppose timeout=None is equivalent to waiting till completion

Reply:
The default is set to 5sec for REST.  So, 300 should do.
```
timeout:
            Timeout for REST and gRPC API requests.
            Default: 5 seconds for REST and unlimited for gRPC
```